### ISSUE TITLE
geo_mask设置的疑问

### ISSUE THREAD

**Issue by Miller-Xu** [BUAABIGSCity/PDFormer#59]
为啥模型中的geo_mask是大于阈值设置为1
```python
sh_mx = sh_mx.T
self.geo_mask = torch.zeros(self.num_nodes, self.num_nodes).to(self.device)
self.geo_mask[sh_mx >= self.far_mask_delta] = 1
self.geo_mask = self.geo_mask.bool()
```
原文写的是
From the short-range view, we define the binary geo-graphic masking matrix Mgeo, and only if the distance (i.e., hops in the graph) between two nodes is less than a threshold λ, the weight is 1, and 0 otherwise.
